README.md

```diff
@@ -99,7 +99,7 @@ This class of models uses audio data to train models that can identify voice, ge
 |Model Class |Reference |Description |
 |-|-|-|
 |Speech recognition with deep recurrent neural networks|	[Graves et al.](https://www.cs.toronto.edu/~fritz/absps/RNN13.pdf)|A RNN model for sequential data for speech recognition. Labels problems where the input-output alignment is unknown<br>[contribute](contribute.md)|
-|Deep voice: Real time neural text to speech |	[Arik et al.](https://arxiv.org/abs/1702.07825)	|A DNN model that performs end-to-end neural speech synthesis. Requires fewer parameters and it is faster than other systems. <br>[contribute](contribute.md)|
+|<b>[Deep voice: Real time neural text to speech](audio/deepvoice3)</b> |	[Arik et al.](https://arxiv.org/abs/1702.07825)	|A DNN model that performs end-to-end neural speech synthesis. Requires fewer parameters and it is faster than other systems. <br>[contribute](contribute.md)|
```

Comment:
nit: A deep learning model that
nit: Requires fewer parameters and is faster
describe which systems it's faster than

Reply:
Also, remove "contribute.md" file.